llvm/test/CodeGen/DirectX/Metadata/empty-valver1.8.ll

```diff
@@ -0,0 +1,32 @@
+; RUN: opt -S -passes="print<dxil-metadata>" -disable-output %s 2>&1 | FileCheck %s --check-prefix=ANALYSIS
+target datalayout = "e-m:e-p:32:32-i1:32-i8:8-i16:16-i32:32-i64:64-f16:16-f32:32-f64:64-n8:16:32:64"
+target triple = "dxilv1.6-unknown-shadermodel6.6-compute"
+
+; ANALYSIS: Shader Model Version : 6.6
+; ANALYSIS-NEXT: DXIL Version : 1.6
+; ANALYSIS-NEXT: Shader Stage : compute
+; ANALYSIS-NEXT: Validator Version : 1.8
+; ANALYSIS-EMPTY:
+
+; Function Attrs: nounwind memory(none)
+define void @main() local_unnamed_addr #0 {
+entry:
+  ret void
+}
+
+attributes #0 = { nounwind memory(none) }
+
+!dx.valver = !{!0}
```

Comment:
all the other test cases stop after `attributes #0` Is there something special about the empty case that makes you need lined 19-32? If not please delete or add checks.

Reply:
> all the other test cases stop after `attributes #0` Is there something special about the empty case that makes you need lined 19-32? If not please delete or add checks.

Deleted unrelated metadata.